curriculum/challenges/english/25-front-end-development/workshop-tailwind-pricing-component/686386e76197458b7dfd52ee.md

```diff
@@ -0,0 +1,91 @@
+---
+id: 686386e76197458b7dfd52ee
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+Inside the `ul`, create two list items. Inside the first list item, create a `span` element and add the HTML entity `&#10003;` as its text. Right after that entity, add the text `Ad-supported streaming`.
+
+Inside the second list item, create a `span` and add the entity `&#10003;`, and after that, add the text `Curated playlists`.
```

Comment:
the directions should be clearer that the tests are looking for campers to add the add supported text and curated playlist text after the span element. that also matches what you wrote in the hints